src/gcc.ts

```diff
@@ -87,8 +91,7 @@ export function distributionUrl(version: string, platform: string): string {
     default:
       throw new Error(`platform ${platform} is not supported`)
   }
-  const parts = version.split('-')
-  if (parts.length !== 3) {
```

Comment:
As the version string now only contains two `-` this needed to be updated as well.

It looks to me that thiss was basically checking if the version inputted by the user was valid, so I've replaced it with a check for the key in the `versions` object.